filament/backend/src/HandleAllocator.cpp

```diff
@@ -74,8 +74,7 @@ HandleBase::HandleId HandleAllocator::allocateHandleSlow(size_t size) noexcept {
 
     if (UTILS_UNLIKELY(id == (HEAP_HANDLE_FLAG|1u))) { // meaning id was zero
         PANIC_LOG("HandleAllocator arena is full, using slower system heap. Please increase "
-                  "FILAMENT_OPENGL_HANDLE_ARENA_SIZE_IN_MB which is currently set to %u MiB.",
-                FILAMENT_OPENGL_HANDLE_ARENA_SIZE_IN_MB);
+                  "the appropriate constant (e.g. FILAMENT_OPENGL_HANDLE_ARENA_SIZE_IN_MB).");
```

Comment:
We should probably rename this to something more generic now that other backends are going to use it, like `FILAMENT_HANDLE_ARENA_SIZE_IN_MB`.

Reply:
Well, Vulkan might need a larger arena size since stuff like its HwRenderPrimitive handle is (by necessity) much larger.  I guess I was thinking of making separate constants for each backend...at least for now.

Reply:
Got it, yeah that makes sense.